src/utils/validation.ts

```diff
@@ -352,8 +352,8 @@ export const validation = {
   },
 
   isValidSharePointUrl(url: string): boolean | string {
-    if (!url) {
-      return false;
+    if (typeof url !== 'string') {
+      return 'URL must be a string';
```

Comment:
To keep it more in line with the other error, I suggest that we rephrase it a bit to "SharePoint Online site URL must be a string."
Also, let's end the error message with a period.
For the other error that is already there, could we add a period there as well? And maybe let's encapsulate the `url` value with single quotes to make it clearer.